autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/deviceauth/AuthLauncher.kt

```diff
@@ -37,12 +36,14 @@ import timber.log.Timber
 
 interface AuthLauncher {
     fun launch(
+        @StringRes featureTitleText: Int,
```

Comment:
We're now using device auth for different purposes (accessing password list + authenticating to confirm delete-all) so need to be able to provide the title dynamically)